vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java

```diff
@@ -846,26 +948,23 @@ public static final class XXLarge {
                 private XXLarge() {
                 }
             }
-
         }
-
     }
 
     /**
      * Classes for defining the height of an element.
      */
     public static final class Height {
 
-        public static final String AUTO = "h-auto";
-        public static final String FULL = "h-full";
```

Comment:
Some changes here are just about moving rows around. IMO it's better to revert those for smaller diff.

Reply:
Yeah, I want them to line up with the utility classes. Easier to keep track.